cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/InstanceTypeDTO.java

```diff
@@ -3,13 +3,17 @@
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 
+import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 @ApiModel( description = "VM instance sizing" )
-@SuppressWarnings( "squid:S2637" )
-public class InstanceTypeDTO extends DTO
+public class InstanceTypeDTO extends NamedEntityDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2160)